packages/browser/package-lock.json

```diff
@@ -8,7 +8,6 @@
       "version": "7.10.4",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.10.4.tgz",
       "integrity": "sha512-vG6SvB6oYEhvgisZNFRmRCUkLz11c7rp+tbNTynGqc6mS1d5ATd/sGyV6W0KZZnXRKMTzZDRgQT3Ou9jhpAfUg==",
-      "dev": true,
```

Comment:
I'm a bit surprised at there being a whole lot of these lines in here. I'm not sure exactly what they mean, but I suppose it might be something like a devDependency suddenly being moved into regular dependencies? Or possible using a different version of npm? Either, just wanted to ask for a double-check that this is still fine.

(I suppose if `npm ci` doesn't fail it should be fine.)

Reply:
Yeah there's something wrong going on with the package-lock... I think the root cause is something that wasn't correctly commited when cutting the release. My intuitiion is that https://github.com/inrupt/solid-client-authn-js/pull/503 would solve it, could you have a look/ approve it so that I can rebase this on it ?